types/g123jp__app-ctr-navbar/index.d.ts

```diff
@@ -0,0 +1,32 @@
+// Type definitions for @g123jp/app-ctr-navbar 0.1
+// Project: https://www.npmjs.com/package/@g123jp/app-ctr-navbar
```

Comment:
Hey, the package is listed on NPM as already shipping TypeScript support:
https://www.npmjs.com/package/@g123jp/app-ctr-navbar
and there are ts files in dist:
https://unpkg.com/browse/@g123jp/app-ctr-navbar@0.1.5/dist/

Reply:
OK, maybe i don't need type any more, thank you